### ISSUE TITLE
How Does This Work (Ethernet.h)?

### ISSUE THREAD

**Issue by thestumbler** [arduino-libraries/Ethernet#237]
There are comments in the file `Ethernet.h` saying that the code no longer uses the two include files `EthernetClient.h` and `EthernetServer.h`. That is true, but it does try to include two flies `Client.h` and `Server.h`, neither of which exist (their prepended-with-Ethernet cousins so exist, but seemingly aren't used).

Am I missing something?

**Comment by per1234** [comment]
Hi @thestumbler. Thanks for your interest in this open source project. This issue tracker is only to be used to report bugs or feature requests specific to the project. This topic is more appropriate for the Arduino Forum. I'm sure we will be able to help you out over there:

https://forum.arduino.cc/

**Comment by thestumbler** [comment]
Let me rephrase it then, as it sure seems to be a bug. 

The main include file `Ethernet.h` tries to include two files which do not exist -- `Client.h` and `Server.h`.

**Comment by per1234** [comment]
It is not a bug. As I told you already, post on the forum and we'll explain things to you.
